src/iac_scan_runner/functionality/scan_runner.py

```diff
@@ -27,14 +27,13 @@
 from iac_scan_runner.checks.shellcheck import ShellCheck
 from iac_scan_runner.checks.snyk import SnykCheck
 from iac_scan_runner.checks.sonar_scanner import SonarScannerCheck
-from iac_scan_runner.checks.steampunk_scanner import SteampunkScannerCheck
+from iac_scan_runner.checks.spotter import SpotterCheck
```

Comment:
```suggestion
from iac_scan_runner.checks.steampunk_spotter import SteampunkSpotterCheck
```

Please, also rename all `spotter` occurrences and files to `steampunk-spotter`/`steampunk_spotter`.